### DISCUSSION TITLE
Display coeffs for exogneous variables

### DISCUSSION THREAD

**Discussion by xeniasaar** [Nixtla/statsforecast#794] [Q&A]
Dear all,

I am using the AutoARIMA model for my master's thesis and added exogenous variables to my model. Is there a way to display the estimated coefficients, standerd error, as well as p- and t-values for the exogenous variables? I need to report them in my thesis. When I just train the model, while adding a X containing the additional variables, the sf.fitted_[0][0].model_ command only displays the info for the ARIMA part of the model. When I follow the how to guide for exogenous regressors, and train and predict in one command, the sf.fitted_[0][0].model_ does not work at all for me.
Can anyone help me?
Thanks in advance!

**Comment by jmoralez** [comment]
Hey @xeniasaar, thanks for using statsforecast. The coefficients are included in the output and are named `ex_*`. Consider the following example:

```python
import numpy as np
from statsforecast import StatsForecast
from statsforecast.models import AutoARIMA
from statsforecast.utils import AirPassengersDF

# without exog
sf = StatsForecast(models=[AutoARIMA(season_length=12)], freq='M')
sf.fit(AirPassengersDF)
print(sf.fitted_[0, 0].model_['coef'], sf.fitted_[0, 0].model_['var_coef'])
# {'ar1': -0.30005076872006264} [[0.00632474]]

# with one exog
df2 = AirPassengersDF.copy()
df2['exog'] = np.random.rand(df2.shape[0])
sf.fit(df2)
print(sf.fitted_[0, 0].model_['coef'], sf.fitted_[0, 0].model_['var_coef'])
# {'ar1': -0.3008079306409614, 'ex_1': 0.5378074605076126} [[0.00631843 0.] [0. 0.00763359]]
```
